pkg/security/tests/activity_tree_test.go

```diff
@@ -0,0 +1,3967 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build functionaltests
```

Comment:
any reason to have it in the functional test suite ? it will be run on all the OS version.

Reply:
agreeing with Sylvain here, if the test does not need a module and is "data-structure and algorithms only" then it should be included in functional tests

Reply:
The reason was to built the exporter test exporter test (which require resolvers). And as it uses the same data as the test itself it was more convenient.
I guess when we'll have a stable merge and insertion algo, with stable tests, we'll could get rid off of the JSON exporter and put back the tests as unitary ones. Does it make sense?

Reply:
Ok I've removed the test definitions as structs to keep only json files, and put back the test suite as unitary one